source/llassetgen-rendering/WindowQt.h

```diff
@@ -53,6 +53,7 @@ class WindowQt : public QWindow {
     virtual void fontColorBChanged(QString value);
     virtual void resetTransform3D();
     virtual void superSamplingChanged(int index);
+    virtual void toggleDistanceField(bool activated);
```

Comment:
Defining this in the base class was necessary due to some Qt weirdness, right?

Reply:
Exactly.